resolvers/ambulance.js

```diff
@@ -4,47 +4,83 @@ const db = require("../models");
 
 const ambulanceResolvers = {
   Query: {
-    ambulances: () => db.ambulance.findAll(),
-    ambulance: (obj, args, context, info) => db.ambulance.findByPk(args.id)
+    ambulances: () =>
+      db.ambulance.findAll({
+        include: [
+          {
+            model: db.event,
+            attributes: [
```

Comment:
is there a reason we need to list all the attributes here (this is done a couple of times)? this example i found seems to only have the model
https://gist.github.com/zcaceres/83b554ee08726a734088d90d455bc566#vanilla-include

Reply:
I had no idea that we could do this. I assume that Sequelize parses this into a `SELECT *`. I removed the attribute list from queries with this similar behaviours.